### PR TITLE
API-2797 Create INSURANCE test request

### PR DESCRIPTION
**Pull request by btomlin**
This request file can be used to insert a new Insurance Company, Group Plan, Coverages and Patient Policy in a VistA Docker test instance in Sandbox. Change your access and verify codes to use the LHS,CONNECTOR PROXY codes.

### CODE REVIEW

File: charon/requests/iblhs-amcms-update-ins-request.json [hunk 1]
```diff
@@ -0,0 +1,185 @@
+{
+   "principal":{
+      "applicationProxyUser":"IBLHS,APPLICATION PROXY",
+      "accessCode":"SECRET",
+      "verifyCode":"SECRET*"
+   },
```
Comment (jhulbert-va) [inline charon/requests/iblhs-amcms-update-ins-request.json:6]: The values in `principal` for sample requests should be bash vars so they can be subsituted.
```
{
  "principal": {
    "applicationProxyUser":"$VISTA_APP_PROXY_USER",
    "accessCode":"$VISTA_APP_PROXY_ACCESS_CODE",
    "verifyCode":"$VISTA_APP_PROXY_VERIFY_CODE"
  }
```

**Review by jhulbert-va — COMMENTED** [review]
In the future, branch names require small descriptions as well (e.g. `API-2797-lh-insurance-request`). The build is failing branch name validation. Bryan or I can merge it regardless, but just an FYI.

File: charon/requests/iblhs-amcms-update-ins-request.json (hunk 1 above)
Comment (btomlin) [reply charon/requests/iblhs-amcms-update-ins-request.json:6]: Thanks Josh, I'll fix this

**Comment by btomlin** [comment]
> In the future, branch names require small descriptions as well (e.g. `API-2797-lh-insurance-request`). The build is failing branch name validation. Bryan or I can merge it regardless, but just an FYI.

Thanks for letting me know about the branch naming. I wasn't aware of that requirement.

**Pull request merged**